src/assets/lang/locale-de.json

```
@@ -135,7 +135,7 @@
 		"itemsPerPage": "Ergebnisse pro Seite",
 		"ofLabel": "von"
 	},
-	"columnSettings": {
+	"tableSettings": {
 		"overlayTitle": "Spalten individualisieren",
```

Damit es konsistent ist:
```suggestion
		"overlayTitle": "Tabelle individualisieren",
```